lib/checks/color/color-contrast.js

```diff
@@ -1,6 +1,12 @@
+var useScroll = true;
+
+if((options || []).indexOf('noScroll') !== -1) {
```

Comment:
@dylanb is this what you had in mind regarding options? The default will be to always scroll, unless the options array includes the string `"noScroll"`.